src/workerd/util/BUILD.bazel

```diff
@@ -195,6 +195,7 @@ wd_cc_library(
     hdrs = ["capnp-mock.h"],
     visibility = ["//visibility:public"],
     deps = [
+        "@capnp-cpp//src/capnp:capnp-rpc",
```

Comment:
hmm, none of the headers of the capnp-rpc are being used by capnp-mock, but some capnp bazel targets are pretty peculiar (getting link errors based on functions being declared in one target but defined in another one, long story that is out of scope here), so if there's a different reason to include this here go ahead.